options.rb

```diff
@@ -0,0 +1,23 @@
+# we will use this file to store our options
+require_relative 'app'
+class Options
+  def option(input)
+    meth = App.new
+    case input
+    when '1'
+      meth.list_all_books
+    when '2'
+      meth.list_all_persons
```

Comment:
- After adding a person to the library, when I try to list all persons, I still get `Please add a person to the library.` Kindly fix this 👍🏽
![Screenshot from 2023-03-13 16-59-12](https://user-images.githubusercontent.com/44624138/224762740-8442ace4-073f-485e-9f68-c77f7759b315.png)